compileAndBuildVLCKit.sh

```diff
@@ -797,38 +800,37 @@ build_universal_static_lib() {
 
     if [ -d ${VLCROOT}/install-"$OSSTYLE"Simulator ];then
         spushd ${VLCROOT}/install-"$OSSTYLE"Simulator
-
-        if (is_simulator_arch $arch);then
-            echo "SIMU OS: $arch"
-            spushd $actual_arch/lib/vlc/plugins
-            for i in `ls *.a`
+            for i in `ls .`
```

Comment:
Why drop the .a check?

Reply:
It shouldn't have, it's just the block moving up.